Modules/Loadable/Volumes/qSlicerVolumesReader.cxx

```diff
@@ -173,9 +173,9 @@ bool qSlicerVolumesReader::load(const IOProperties& properties)
     fileList.GetPointer());
   if (node)
     {
-    if (properties.contains("colorNodeID"))
+    QString colorNodeID = properties.value("colorNodeID", QString()).toString();
```

Comment:
I suggest to update the commit message to also indicate that the volume reader logic checking `colorNodeID` has been improved.

Reply:
OK!